SCP -> BTP, Cloud Foundry - Cloud Foundry environment (#199)
SAP/cloud-sdk: docs/java/guides/linux-how-to.mdx

To find out more about Apache Maven, how to get it for your Linux distribution, and detailed tutorials [follow the official maven documentation](https://maven.apache.org/).

> ### Cloud Foundry environment Command Line Interface

Author: ```suggestion
### Cloud Foundry Environment Command Line Interface
```

After a successful build, you'll find a `target` folder within your app's root folder containing a `war` file along with other build artifacts.
This `war` file is a packaged version of your web-app that's going to be deployed in the SAP Cloud Foundry environment.

> ### Bind your Command-Line Interface to SAP Cloud Foundry environment

Author: ```suggestion
### Bind your Command-Line Interface to the SAP BTP Cloud Foundry environment
```

    cf api https://api.cf.eu10.hana.ondemand.com
```

> Provide your credential for SAP Cloud Foundry environment by running:

Author: ```suggestion
Provide your credentials for the SAP BTP Cloud Foundry environment by running:
```
